lib/cli/src/c_gen/staticlib_header.rs

```diff
@@ -5,71 +5,75 @@ use wasmer_types::ModuleInfo;
 use wasmer_types::{Symbol, SymbolRegistry};
 
 /// Helper functions to simplify the usage of the static artifact.
-const HELPER_FUNCTIONS: &str = r#"
-wasm_byte_vec_t generate_serialized_data() {
+fn gen_helper_functions(atom_name: &str, module_name: &str) -> String {
```

Comment:
No, not a good idea. It's better to make the CI fail in that case, so that we know when to use `atom-name` vs `atom_name` at the call site, not here. This would be defensive coding, which I don't like.